docs/index.md

```diff
@@ -23,42 +25,3 @@ Charmed MySQL K8s is an official distribution of MySQL. It’s an open-source pr
-| 2     | t-enable-security             | [7. Enable security](/t/charmed-mysql-k8s-tutorial-enable-security/9669)                       |
-| 2     | t-cleanup-environment         | [8. Cleanup your environment](/t/charmed-mysql-k8s-tutorial-cleanup-environment/9665)          |
-| 1     | how-to                        | [How To]()                                                                                     |
-| 2     | h-manage-units                | [Manage units](/t/charmed-mysql-k8s-how-to-manage-units/9659)                                  |
-| 2     | h-enable-encryption           | [Enable encryption](/t/charmed-mysql-k8s-how-to-enable-encryption/9655)                        |
-| 2     | h-manage-app                  | [Manage applications](/t/charmed-mysql-k8s-how-to-manage-app/9657)                             |
-| 2     | h-configure-s3-aws               | [Configure S3 AWS](/t/charmed-mysql-k8s-how-to-configure-s3-for-aws/9651)                                  |
-| 2     | h-configure-s3-radosgw                | [Configure S3 RadosGW](/t/charmed-mysql-k8s-how-to-configure-s3-for-radosgw/10319)                                  |
-| 2     | h-create-and-list-backups     | [Create and List Backups](/t/charmed-mysql-k8s-how-to-create-and-list-backups/9653)            |
-| 2     | h-restore-backup              | [Restore a Backup](/t/charmed-mysql-k8s-how-to-restore-backup/9663)                            |
-| 2     | h-migrate-cluster-via-restore | [Cluster Migration with Restore](/t/charmed-mysql-k8s-how-to-migrate-cluster-via-restore/9661) |
-| 2     | h-enable-monitoring           | [Enable Monitoring](/t/charmed-mysql-k8s-how-to-enable-monitoring/9981)                        |
-| 1     | reference                     | [Reference]()                                                                                  |
-| 2     | r-actions                     | [Actions](https://charmhub.io/mysql-k8s/actions)                                               |
-| 2     | r-configurations              | [Configurations](https://charmhub.io/mysql-k8s/configure)                                      |
-| 2     | r-libraries                   | [Libraries](https://charmhub.io/mysql-k8s/libraries/helpers)                                   |
-| 2     | r-integrations                   | [Integrations](https://charmhub.io/mysql-k8s/integrations)                                   |
-| 1     | explanation                    | [Explanation]()                                                                                                      |
-| 2     | e-interfaces                | [ Interfaces/endpoints](/t/charmed-mysql-k8s-explanations-interfaces-endpoints/10249) |
-| 2     | e-flowcharts                | [ Charm flowcharts](/t/charmed-mysql-k8s-explanation-charm-lifecycle-flowcharts/10031) |
-
-# Redirects
-
-[details=Mapping table]
-| Path | Location |
-| ---- | -------- |
-[/details]
```

Comment:
@deusebio please confirm, it is expected to remove this section from GitHub.
P.S. personally I was find it useful. Also, in case if we lost Discourse, how to restore the content from GitHub without navtable? Tnx!

Reply:
we provide a single point of entry (which is the docs key in the metadata.yaml) and there one should have all relevant topic. The tree structure is created by parsing and navigating the navigation table. So NavTable only exists on Discourse. This allow us to centralize this information and have a single index.